lib/operators/increase-operators.coffee

```diff
@@ -1,16 +1,17 @@
 {Operator} = require './general-operators'
 {Range} = require 'atom'
+settings = require '../settings'
 
 #
 # It increases or decreases the next number on the line
 #
 class Increase extends Operator
   step: 1
-  numberRegex: new RegExp("-?[0-9]+")
 
   constructor: ->
     super
     @complete = true
+    @numberRegex = new RegExp(settings.numberRegex() or settings.config.numberRegex.default)
```

Comment:
I think you don't need to explicitly fall back on the default value; [`atom.config.get`](https://atom.io/docs/api/v0.196.0/Config#instance-get) should return the default value if the user has not overridden it.